### PR TITLE
mlbdd.0.4 - via opam-publish

### PR DESCRIPTION
**Pull request by arlencox**
A high performance BDD implementation in OCaml

The mlbdd library provides a simple, easy-to-use, easy-to-extend implementation of binary decision diagrams (BDDs) in OCaml. It is well tested and well documented. The library itself has no dependencies and is thus easy to include in applications that might, for example, be compiled with js_of_ocaml or other tools that rely on pure OCaml. It is also easier to integrate with existing projects due to its lack of dependencies.

Critically, this BDD implementation uses a garbage-collection-aware hashing scheme, so that unused nodes can be collected.  Additionally, this implementation uses complement edges to significantly improve performance over the simplest BDD implementations.

---
* Homepage: https://github.com/arlencox/mlbdd
* Source repo: https://github.com/arlencox/mlbdd.git
* Bug tracker: https://github.com/arlencox/mlbdd/issues

---
Pull-request generated by opam-publish v0.2.1

### CODE REVIEW

**Comment by UnixJunkie** [comment]
interesting travis error:
Fatal error in module common/cudfSolver.ml

**Comment by arlencox** [comment]
I have no idea what this error means.  There are no new modules or anything that might affect a build.  The changes consist of the following:

1. A single added api function
2. A bug fix in an existing api function
3. A new test corresponding to that bug

File: packages/mlbdd/mlbdd.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+maintainer: "Arlen Cox <arlencox@gmail.com>"
+authors: "Arlen Cox <arlencox@gmail.com>"
+homepage: "https://github.com/arlencox/mlbdd"
+bug-reports: "https://github.com/arlencox/mlbdd/issues"
+license: "MIT"
+dev-repo: "https://github.com/arlencox/mlbdd.git"
+build: [make]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "mlbdd"]
+depends: "ocamlfind" {build}
```
Comment (samoht) [inline packages/mlbdd/mlbdd.0.4/opam]: should be `[ "ocamlfind" {build} ]`

**Comment by samoht** [comment]
I've re-run the build. The error is hopefully transient (but sometimes it indicates something wrong with opam metadata ... in that case I don't see why it's the case only on one test).

File: packages/mlbdd/mlbdd.0.4/opam (hunk 1 above)
Comment (arlencox) [inline packages/mlbdd/mlbdd.0.4/opam]: I'll correct this.

**Comment by UnixJunkie** [comment]
it's all green now except for mac

**Comment by arlencox** [comment]
The problem on mac is silly.  I don't understand what the problem is.

```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - mlbdd.0.4
          - ocamlfind.1.5.5
        (This might be due to outdated metadata, in this case run 'opam
        update')
```

**Comment by samoht** [comment]
See https://github.com/ocaml/opam-repository/pull/4139 for the OSX breakage reason.

**Pull request merged**